src/smt/theory_str_noodler/theory_str_noodler.cpp

```diff
@@ -2062,9 +2116,10 @@ namespace smt::noodler {
      * @param len_formula Formula to be check
      * @return lbool Sat
      */
-    lbool theory_str_noodler::check_len_sat(expr_ref len_formula) {
+    lbool theory_str_noodler::check_len_sat(expr_ref len_formula, model_ref &mod) {
         int_expr_solver m_int_solver(get_manager(), get_context().get_fparams());
         m_int_solver.initialize(get_context());
-        return m_int_solver.check_sat(len_formula);
+        auto ret = m_int_solver.check_sat(len_formula);
+        return ret;
```

Comment:
There is no reason for this change, is there?

Reply:
Probably not